app/controllers/v0/profile/direct_deposits_controller.rb

```diff
@@ -44,6 +50,15 @@ def update
 
       private
 
+      def log_control_info(control_info, action)
+        ::Rails.logger.info("Direct Deposit Control Info: #{action}",
+                            { benefit_type: control_info.benefit_type,
+                              updatable: control_info.account_updatable?,
+                              valid: control_info.valid?,
+                              restrictions: control_info.restrictions.join(', '),
+                              errors: control_info.errors.join(', ') })
```

Comment:
It doesn't look like it, but I want to be sure... is there any PII in here? (I see "direct deposit" and I worry 😬)

Reply:
@rmtolmach No PII.  Thank you for checking though.  The control info fields are just boolean flags related to the user and account.   This extra logging is so we can monitor activity around some changes to direct deposit.  It'll most likely be temporary.